crates/interpreter/src/instruction_result.rs

```diff
@@ -24,6 +24,7 @@ pub enum InstructionResult {
     OpcodeNotFound,
     CallNotAllowedInsideStatic,
     StateChangeDuringStaticCall,
+    ActiveAccountUnsetAuthCall,
```

Comment:
```suggestion
    UnauthorizedAuthCall,
```
just a proposal, may read a bit easier